openmetadata-service/src/test/java/org/openmetadata/service/alerts/AlertsRuleEvaluatorTest.java

```diff
@@ -3,138 +3,165 @@
 import java.util.UUID;
 import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.Test;
-import org.mockito.Mockito;
+import org.junit.jupiter.api.TestInfo;
+import org.openmetadata.schema.api.data.CreateTable;
 import org.openmetadata.schema.entity.data.Table;
-import org.openmetadata.schema.entity.teams.Team;
-import org.openmetadata.schema.entity.teams.User;
+import org.openmetadata.schema.tests.type.TestCaseResult;
+import org.openmetadata.schema.tests.type.TestCaseStatus;
+import org.openmetadata.schema.type.ChangeDescription;
 import org.openmetadata.schema.type.ChangeEvent;
-import org.openmetadata.schema.type.EntityReference;
-import org.openmetadata.schema.type.TagLabel;
+import org.openmetadata.schema.type.Column;
+import org.openmetadata.schema.type.ColumnDataType;
+import org.openmetadata.schema.type.EventType;
+import org.openmetadata.schema.type.FieldChange;
 import org.openmetadata.service.Entity;
-import org.openmetadata.service.jdbi3.CollectionDAO.TeamDAO;
-import org.openmetadata.service.jdbi3.CollectionDAO.UserDAO;
-import org.openmetadata.service.jdbi3.TableRepository;
-import org.openmetadata.service.jdbi3.TeamRepository;
-import org.openmetadata.service.jdbi3.UserRepository;
-import org.openmetadata.service.security.policyevaluator.ResourceContext;
+import org.openmetadata.service.OpenMetadataApplicationTest;
+import org.openmetadata.service.resources.EntityResourceTest;
+import org.openmetadata.service.resources.databases.TableResourceTest;
 import org.springframework.expression.EvaluationContext;
+import org.springframework.expression.spel.support.StandardEvaluationContext;
 
-class AlertsRuleEvaluatorTest {
-  private static Table table;
-  private static User user;
-  private static EvaluationContext evaluationContext;
+class AlertsRuleEvaluatorTest extends OpenMetadataApplicationTest {
+  private static TableResourceTest tableResourceTest;
 
   @BeforeAll
-  public static void setup() {
-    Entity.registerEntity(User.class, Entity.USER, Mockito.mock(UserDAO.class), Mockito.mock(UserRepository.class));
-    Entity.registerEntity(Team.class, Entity.TEAM, Mockito.mock(TeamDAO.class), Mockito.mock(TeamRepository.class));
-    table = new Table().withName("table");
-    user = new User().withId(UUID.randomUUID()).withName("user");
-    ResourceContext resourceContext =
-        ResourceContext.builder()
-            .resource("table")
-            .entity(table)
-            .entityRepository(Mockito.mock(TableRepository.class))
-            .build();
-    // SubjectContext subjectContext = new SubjectContext(user);
-    // RuleEvaluator ruleEvaluator = new RuleEvaluator(null, subjectContext, resourceContext);
-    ChangeEvent event = new ChangeEvent();
-    // evaluationContext = new StandardEvaluationContext(ruleEvaluator);
+  public static void setup(TestInfo test) throws URISyntaxException, IOException {
+    tableResourceTest = new TableResourceTest();
+    tableResourceTest.setup(test);
```

Comment:
This will make this test non-unit test, right? When you set up tableResourceTest, you need to start a cluster? Can this be turned into unit test similar to RuleEvaluatorTest?